src/Cake.Core/IO/ProcessRunner.cs

```diff
@@ -69,18 +69,19 @@ public IProcess Start(FilePath filePath, ProcessSettings settings)
 
             // Get the working directory.
             var workingDirectory = settings.WorkingDirectory ?? _environment.WorkingDirectory;
```

Comment:
This could be changed to `var workingDirectory = settings.WorkingDirectory?.MakeAbsolute(_environment)`.